addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -1206,14 +1206,9 @@
             <field name="name">Expense Reports to Approve</field>
             <field name="res_model">hr.expense.sheet</field>
             <field name="view_mode">tree,kanban,form,pivot,graph</field>
-            <field name="search_view_id" ref="hr_expense_sheet_view_search"/>
-            <field name="context">{
-                'search_default_submitted': 1,
-                'search_default_department_id': [active_id],
-                'default_department_id': active_id
-                }
-            </field>
-            <field name="domain">[('state', '=', 'submit'), ('department_id', '=', active_id)]</field>
```

Comment:
hum, I am wondering, whether to leave domain for `department_id` or add `department_id` in the searchpanel..

I guess I would rather leave domain.

Thoughts @JulienAlardot? :thinking: